api/src/services/assessmentsDummyService.ts

```diff
@@ -0,0 +1,180 @@
+import db from './db';
+
+interface ProgramParticipantsRow {
+  id: number;
+  principal_id?: number;
+  program_id?: number;
+  role_id?: number;
+}
+
+/**
+ * return matching rows from the program_participants table that have  been inserted or updated based on their pre-existence in the table.
```

Comment:
Capitalize the beginning of each comment if you can, also press return to shorten the length of the line to below 80 characters